### ISSUE TITLE
`minikube delete` command crashes when minikube is not up

### ISSUE THREAD

**Issue by gtirloni** [kubernetes/minikube#1280]
**Is this a BUG REPORT or FEATURE REQUEST?**: Bug Report
**Minikube version**: 0.17.1
**Environment**:
- **OS**: Fedora 25
- **VM Driver**: VirtualBox 5.1.18
- **ISO version**: v1.0.7

**What happened**:

Running `minikube delete` when minikube is not up causes an error and it attempts to report crash information:

```
$ minikube delete
Deleting local Kubernetes cluster...
Errors occurred deleting machine:  Error deleting host: minikube: Host does not exist: "minikube"
================================================================================
An error has occurred. Would you like to opt in to sending anonymized crash
information to minikube to help prevent future errors?
To opt out of these messages, run the command:
	minikube config set WantReportErrorPrompt false
================================================================================
Please enter your response [Y/n]:
```

**What you expected to happen**:
A simple message stating minikube is not running would suffice.

**How to reproduce it** (as minimally and precisely as possible):

 1. Ensure minikube VM does not exist
 2. Run `minikube delete`

**Comment by r2d4** [comment]
Yeah this isn't really a "reportable error".  We shouldn't prompt the user here.

**Comment by aaron-prindle** [comment]
Fixed with #1297.  Closing.
